Taxes/Operation/01_ManageDecisionMatrix.md

```diff
@@ -0,0 +1,251 @@
+
+7. Click the *Tax zone* drop-down list and select the applicable tax zone.
+
+8. Click the [SAVE] button.  
+A pop-up window confirms that the decision matrix has been saved. The new entry has been saved.
+
+  ![Decision matrix saved](../../Assets/Screenshots/Taxes/Settings/DecisionMatrix/DecisionMatrixSaved.png "[Decision matrix saved]")
+
+9. Repeat this procedure to configure one by one all the applicable tax scenarios depending on your business needs.
+
+10. If it is necessary to change the position of an entry in the decision matrix, see [Change the order of entries](#change-the-order-of-entries).
+
+  > [Info] It is important to remember that the correct order of the decision-making criteria is crucial, as the system goes through the criteria in the order they are listed.
+
+
+## Define tax exceptions on ZIP code level
+
+Some areas within a territory have special tax regulations. This is the case, for example, of Helgoland in Germany or Northern Ireland in the United Kingdom.
+
+These tax exceptions can be defined using the *Ship-to postcode* fields.
+
+#### Prerequisites  
+
+- An applicable tax class has been created, see [Create a tax class](../Integration/02_ManageTaxClasses.md#create-a-tax-class).
+- An applicable tax zone has been created, see [Create a tax zone](../Integration/03_ManageTaxZones.md#create-a-tax-zone).
+- An applicable tax rate has been created, see [Create a tax rate](../Integration/01_ManageTaxRates.md#create-a-tax-rate).
+
+#### Procedures
+
+*Taxes > Settings > Tab DECISION MATRIX*
+
+![Decision matrix](../../Assets/Screenshots/Taxes/Settings/DecisionMatrix/DecisionMatrix.png "[Decision matrix]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the upper left corner.   
+  A new data entry line is displayed.
+
+2. Click the *VAT ID* drop-down list and select the appropriate option. The following options are available:
+
+  - **Yes**  
+  The end customer is a company and, therefore, has a VAT ID number (business-to-business transaction).
+  - **No**  
+  The end customer is a private individual and, therefore, has no VAT ID number (business-to-customer transaction).
+
+
+3. Click the *Country of origin* drop-down list and select the appropriate option. Use the drop-down list on the left to select a group of countries or the one on the right to select an individual country. As a group of countries, only the **EU - European Union** option is available. In the drop-down list of individual countries, all available countries in the system are displayed. When a group of countries is selected, the drop-down list on the right is locked.
+
+4. Click the *Ship-to country* drop-down lists and select **Any** on the left and the country where the product is to be shipped to on the right.  
```

Comment:
vgl. in procedure davor: **Any** kann ja nicht richtig gewählt werden. Wie in vorheriger Prozedur beschreiben